mobile/src/full/res/values/strings.xml

```diff
@@ -3,6 +3,6 @@
     <string name="info_openhab_gcm_in_progress">Device registration is in progress</string>
     <string name="info_openhab_gcm_failed">Push notification registration failed</string>
     <string name="info_openhab_gcm_failed_play_services">Push notification registration failed due to a Google Play Services issue: %s</string>
-    <string name="info_openhab_gcm_connected">Device successfully registered with Firebase Cloud Messaging</string>
+    <string name="info_openhab_gcm_connected">Notifications are enabled</string>
```

Comment:
This seems a bit overly generic. How about `Successfully registered for notification reception` or `Notification registration completed successfully`?

Reply:
I updated these strings to `Notification notification...`.